### DISCUSSION TITLE
Allow players play singleplayer when waiting in multiplayer room

### DISCUSSION THREAD

**Discussion by LIII-Works** [ppy/osu#16002] [Ideas]
As the title said, sometimes in multiplayer, there are always some people who have slow internet and do not have the map or just waiting for players to join. It costs MUCH MORE time than expected. So if singleplayer is available during waiting in multiplayer room will be nice. shrink multiplayer into a small half-transparent window floating on screen or make multiplayer into a tab. Set a few second countdown for exiting singleplayer session after all players are ready and the host pressed start.

**Comment by bdach** [comment]
dunno if i agree with this. what if you have a good single player play going when the lobby starts?

**Reply by LIII-Works** [reply]
I believe player should know their play will be interrupted by multiplayer. you concern this because you know your player might be interrupted right? we can also add a warning for this after the epilepsy warning in the beginning of a beatmap, so players are expected for this
